<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolPane</class>
 <widget class="QWidget" name="ToolPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pencilButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eraserButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="fillButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string>Fill Bucket</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="redLabel">
     <property name="text">
      <string>Red Value (0-255):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="redSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="greenLabel">
     <property name="text">
      <string>Green Value (0-255):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="greenSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="blueLabel">
     <property name="text">
      <string>Blue Value (0-255):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="blueSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="opacityLabel">
     <property name="text">
      <string>Alpha Value (0-255):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="opacitySlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
